packages/core/docs/commercetools/enterprise/use-review.md

```diff
@@ -0,0 +1,65 @@
+---
+platform: Commercetools
+---
+
+<IncludeContent content-key="use-review" />
+
+::: slot search-params
+
+```typescript
+interface ReviewSearchParams {
+  productId: string;
+  limit?: number;
+  offset?: number;
+}
+```
+
+:::
+
+::: slot add-params
+
+```typescript
+interface ReviewAddParams {
+  productId: string;
+  limit?: number;
+  offset?: number;
+  draft: ReviewDraft;
+}
+
+interface ReviewDraft {
+  authorName: string;
+  text: string;
+  rating: number;
+}
+```
+
+:::
+
+::: slot usage
+
+## Usage
+
+When `@vsf-enterprise/ct-reviews` plugin is installed as a dependency, it requires minor modifications in the code to work.
+
+At the top of `script` tag in `Product.vue`, we're importing `useReview` and `reviewGetters` from `@vue-storefront/commercetools`. To use this plugin, simple import them from `@vsf-enterprise/ct-reviews` instead of `@vue-storefront/commercetools`:
```

Comment:
```suggestion
In the codebase we're importing `useReview` and `reviewGetters` from `@vue-storefront/commercetools`. To use this plugin, simply import them from `@vsf-enterprise/ct-reviews` instead of `@vue-storefront/commercetools`:
```

I changed to something that won't get outdated too quick :)